docker-linux/run_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euo
+ip addr add 192.168.0.6/24 dev lo || true
+ip addr add 192.168.0.7/24 dev lo || true
+ip addr add 192.168.0.8/24 dev lo || true
+ip route add 224.0.0.0/4 dev lo || true
+ip route add 239.255.0.0/8 dev lo || true
+
+ip link set dev lo multicast on
+
+ip a show dev lo
+
+echo "net.ipv4.conf.all.rp_filter = 0" > /etc/sysctl.conf
+echo "net.ipv4.conf.default.rp_filter = 0" >> /etc/sysctl.conf
+echo "net.ipv4.conf.default.rp_filter = 0" >> /etc/sysctl.conf
+echo "net.ipv4.conf.all.accept_local = 1" >> /etc/sysctl.conf
+echo "net.ipv4.conf.all.mc_forwarding=1" >> /etc/sysctl.conf
```

Comment:
I doubt the base image has an sysctl paramers set but in case we change the base image later to one that does, generally best to append to sysctl.conf instead of overwriting it:

```suggestion
echo "net.ipv4.conf.all.rp_filter = 0" >> /etc/sysctl.conf
echo "net.ipv4.conf.default.rp_filter = 0" >> /etc/sysctl.conf
echo "net.ipv4.conf.default.rp_filter = 0" >> /etc/sysctl.conf
echo "net.ipv4.conf.all.accept_local = 1" >> /etc/sysctl.conf
echo "net.ipv4.conf.all.mc_forwarding=1" >> /etc/sysctl.conf
```

We can also merge all of these into a single command if you'd like:

```suggestion
cat << EOF > /etc/sysctl.conf
net.ipv4.conf.all.rp_filter = 0
net.ipv4.conf.default.rp_filter = 0
net.ipv4.conf.default.rp_filter = 0
net.ipv4.conf.all.accept_local = 1
net.ipv4.conf.all.mc_forwarding=1
EOF
```